website/frontend/src/components/blocks/ProgramSwitch/ProgramSwitch.tsx

```diff
@@ -27,12 +27,12 @@ export const ProgramSwitch: VFC<Props> = ({ pageType }) => {
         throw new Error(`WALLET NOT CONNECTED`);
       }
 
-      const response = await apiRequest.getResource(RPC_METHODS.GET_TEST_BALANCE, {
+      const response = await apiRequest.callRPC(RPC_METHODS.GET_TEST_BALANCE, {
         address: `${currentAccount.address}`,
       });
 
       if (response.error) {
-        alert.error(`${response.error.error}`);
```

Comment:
Is there a way to reach to this line? If there will be an error in the response, `callRpc` method will throw an error, I suppose